content/blog/v1-33-released.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Scratch Addons v1.33 is here!
+description: Badges for project creators in comments, a keyboard shortcut for pausing, and more!
+date: 2023-06-09T00:00:00Z
+author: DNin01
+---
+
+Welcome back! It's about time for another Scratch Addons update, so let's go over what we have added in version 1.33!
+
+## See the VIP in an instant
```

Comment:
Why "VIP"

Reply:
And considering Scratch's remix culture, "VIP" is an incorrect term.

Reply:
```suggestion
## Always see the creator in the crowd
```